### ISSUE TITLE
Screen flickers

### ISSUE THREAD

**Issue by alxli** [PFCKrutonium/Windows-10-Login-Background-Changer#133]
It just flickers frantically at the lockscreen. My system is running Windows 10 Enterprise N LTSB with a HiDPI screen (3200 by 1800) and the image I used was this: https://drslash.com/wp-content/uploads/2014/07/bg-5120x2880-1.png

**Comment by anamritraj** [comment]
I had the exact same issue 20 minutes ago.

What you need to do is to replace the file Windows.UI.Logon.pri with Windows.UI.Logon.pri.bak

You can do this anyway you like:
- By logging in into a secondary OS (Ubuntu etc.)- Navigating manually to C:\SystemResources\Windows.UI.Logon and replace the file Windows.UI.Logon.pri with Windows.UI.Logon.pri.bak.
- By making a recovery disk - You can download the Windows 10 iso from Windows website (Google it) and make a Bootable Pen drive. Then Boot from it and go to command promt which will be found somewhere in repair my system.
  - skip the following step and move to next step after it.

-SKIP THIS IF USING ABOVE METHOD
If you can't download the iso, then there is another possible solution, IT MAY NOT WORK FOR YOU.
- Force Shutdown your PC by holding power key for 5-10 seconds.
- Start it, while it is starting, When the windows logo apperars, again force shutdowm your pc.
- Now Start again. This time it will take you to startup repair tool. From there goto advanced settings or advanced options until you see command promt.
  - once you are in command prompt enter the codes below:

```
c:
cd SystemResources\Windows.UI.Logon
Copy Windows.UI.Logon.pri.bak Windows.UI.Logon.pri
y
exit
```

**Comment by alxli** [comment]
I already fixed the problem that way before I made the post. The README just told me to report the bug, so I did in case it might be helpful to know that the issue occurs with my specific image/build/screen resolution. :P

**Comment by darth62969** [comment]
this bug has been reported several times, and we are working to rectify this issue.you can go to the gitter room and see the current progress on this issue and others.

**Comment by akamarvin** [comment]
if it helps, it happened to me too 

from an upgraded 8.1 pro 64, locale be-fr. 
- then the automatic MS updates happened , 
- manual remove and replace of my Nvidia Drivers ( I had huge framerate issues) 
- successful replacement of my lock screen image with a 1920*1200 png,
- then I tried your tool (1.0.2.0), with a solid dark gray color (to match the background of the lock screen). And I applied the change. 
  Tried the lock + login => the "Flicker of hell" began immediately between the lock screen image and the current accent color (dark orange) of my windows theme. And stayed after reboot.
  the Shift-F10 and cmd fix saved the situation. 

I saved the faulty .pri. Would it be of any help to you ? is it safe to share it with you, security wise ? I mean does this .pri file contains any sensitive logon information ?
